tests/websocket.html

```diff
@@ -3,150 +3,215 @@
 	<head>
 		<title>WebSocket example</title>
 		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="https://unpkg.com/purecss@2.0.6/build/pure-min.css" integrity="sha384-Uu6IeWbM+gzNVXJcM9XV3SohHtmWE+3VGi496jvgX1jyvDTXfdK+rfZc8C1Aehk5" crossorigin="anonymous">
 		<style type="text/css">
-
-			body {
-				width: 800px;
-				margin: auto;
-			}
-
-			header {
-				font-size: 36pt;
-				width: 100%;
+			h1, h3 {
 				text-align: center;
-				margin-bottom: 1em;
-				color: #541F14;
-			}
-
-			section.proto {
-				float: left;
-				/*background-color: #f8f8f8;*/
-			}
-
-			section#json {
-				width: 380px;
-				margin-right: 20px;
-			}
-
-			section#raw {
-				width: 380px;
-				padding-left: 16px;
-				border-left: 4px solid #626266;
 			}
 
-			div.desc {
-				margin: 0px;
-			}
 			pre.sent, pre.received {
 				margin-top: 0px;
 				border-radius: 4px;
 				padding: 5px;
 			}
+
 			pre.sent {
-				border: 1px solid #938172;
+				border: 1px solid #4b63cc;
 				background-color: white;
 			}
+
 			pre.received {
-				border: 1px solid #CC9E61;
+				border: 1px solid #4db96d;
 				text-align: right;
 			}
 
+			.ws-state {
+				font-weight: bold;
+				line-height: 18px;
+				vertical-align: middle;
+			}
+
+			div.log {
+				font-size: 13pt;
+			}
 		</style>
 	</head>
 
 	<body>
-		<header>Webdis with HTML5 WebSockets</header>
-
-		<section class="proto" id="json">
-			<h3>JSON</h3>
-			<div class="log" id="json-log">
-				Connecting...
+		<div class="pure-g">
+		<h1 class="pure-u-1">Webdis with HTML5 WebSockets</h1>
+		</div>
+
+		<div class="pure-g">
+			<div class="pure-u-1-8"></div>
+			<div class="pure-u-1-3">
+				<h3>JSON</h3>
+				<form class="pure-form">
+					<fieldset>
+						<div class="pure-g">
+							<div class="pure-u-2-3"><label class="ws-state pure-u-23-24" id="json-state">State: Disconnected</label></div>
+							<div class="pure-u-1-3"><button type="submit" class="pure-u-23-24 pure-button pure-button-primary" id="json-btn-connect">Connect</button></div>
```

Comment:
what's the point of having 2 levels of nesting here? 2/3 × 23/24… is it for spacing? doesn't PureCSS already include spaces between elements?
In this domain I've only used [Skeleton.css](https://getskeleton.com/), and it did include some spaces between elements.

Reply:
Without this 1/24th grid spacing, the elements all touch. I don't think the extra tags are a major problem in this file that had not been touched for years, it's not frequently edited.